app/controllers/admin/campaigns_controller.rb

```diff
@@ -1,6 +1,7 @@
 class Admin::CampaignsController < AdminController
   before_action :set_campaign, only: [:show, :edit, :update, :destroy]
   helper_method :sort_column, :sort_direction
+  before_filter :authorize_admin
```

Comment:
`before_filter` is the old syntax for this. The new one is `before_action`

As we are using this method (authorize_admin) in most admin controllers the best is to set the `before_action` on the AdminController and then use `skip_before_action :authorize_admin` on the UsersController where you don't require that level of authorization.